src/globus_cli/commands/flows/update.py

```diff
@@ -0,0 +1,210 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+
+from globus_cli.commands.flows._common import (
+    administrators_option,
+    description_option,
+    input_schema_option,
+    keywords_option,
+    starters_option,
+    subtitle_option,
+    viewers_option,
+)
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import (
+    JSONStringOrFile,
+    MutexInfo,
+    ParsedJSONData,
+    command,
+    flow_id_arg,
+    mutex_option_group,
+)
+from globus_cli.termio import Field, TextMode, display, formatters
+from globus_cli.types import JsonValue
+
+ROLE_TYPES = ("flow_viewer", "flow_starter", "flow_administrator", "flow_owner")
+
+
+@command("update", short_help="Update a flow")
+@flow_id_arg
+@click.option("--title", type=str, help="The name of the flow.")
+@click.option(
+    "--definition",
+    type=JSONStringOrFile(),
+    help="""
+        The JSON document that defines the flow's instructions.
+
+        The definition document may be specified inline, or it may be
+        a path to a JSON file.
+
+            Example: Inline JSON:
+
+            \b
+            --definition '{{"StartAt": "a", "States": {{"a": {{"Type": "Pass", "End": true}}}}}}'
+
+            Example: Path to JSON file:
+
+            \b
+            --definition definition.json
+    """,  # noqa: E501
+)
+@click.option(
+    "--owner",
+    type=str,
+    help="""
+        Assign ownership to your Globus Auth principal ID.
+
+        This option can only be used to take ownership of a flow,
+        and your Globus Auth principal ID must already be a flow administrator.
+
+        This option cannot currently be used to assign ownership to an arbitrary user.
+    """,
+)
+@subtitle_option
+@description_option
+@input_schema_option
+# "--no-Xs" options must exist to clear lists of things (like administrators).
+# These must be mutually exclusive with the flags that set options.
+@administrators_option
+@click.option(
+    "--no-administrators",
```

Comment:
I think `no-` options are probably fine, but would it be clearer if we went with something like `clear-`, `reset-`, or `erase-`?

Reply:
As discussed, we'll target a comma-separated list of strings, and allow an empty string to act as the sentinel value for erasure.